game/src/sandbox/dashboards/commuter.rs

```diff
@@ -275,6 +276,33 @@ impl State for CommuterPatterns {
                     }
                 }
 
+                batch.push(Color::BLACK.alpha(0.5), block.shape.clone());
```

Comment:
> It'd be a bit more clear to do this above the others block, right after for i in &block.borders { batch.push(Color::PURPLE, app.primary.map.get_i(*i).polygon.clone()); }, since it deals with drawing the current block.

Done.